README.md

```diff
@@ -94,6 +94,45 @@ export function App() {
 
 > Update the version in the script tag above with the exact version of this package you would like to use.
 
+#### Web component attributes and properties
+
+Each React component is defined as a custom element with a matching name in kebab-case.
```

Comment:
The formatting and phrasing of this reads like a list—is it supposed to be a list?